components/browser/containers/src/main/java/mozilla/components/browser/containers/Container.kt

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.browser.containers
+
+/**
+ * A container also known as a contextual identity.
+ */
+interface Container {
+    /**
+     * The cookie store ID for the container.
+     */
+    val cookieStoreId: String
```

Comment:
Is that the id for the context? If yes, should we name this `contextId` like we do in `Session` etc.?

Reply:
Yes `cookieStoreId` is equivalent to `contextId`. There are just a lot of different naming conventions when you look at the GV naming and compare it to the public documentations of Contextual Identities.

I don't have any strong preferences since it does make sense to align with what we have already named it internally.